src/components/mortgage-investment-compare/amortization-table.js

```diff
@@ -11,6 +11,24 @@ const amortizationTable = css`
     font-size: 1rem;
   }
 
+  thead,
+  tbody tr {
+    display: table;
+    width: 100%;
+    table-layout: fixed; /* even columns width , fix width of table too*/
+  }
```

Comment:
sug: Could these be done w/ emotion composition and styled components?

Reply:
I think we would be able to but it could make things worser because there's a lot of shared styling that i'd have to separate apart.  For instance, say I created a THead, TBody, and Tr styled components. They all share the following css as shown above:
```

{
    display: table;
    width: 100%;
    table-layout: fixed; /* even columns width , fix width of table too*/
} 
```
I wouldn't want to duplicate the css for those 3 styled components. The only other option would be to create a class containing that css and to inject the class inside the 3 styled components, but I'm not sure if that's a very clean approach either.

I'll merge the branch and let's keep this convo going on the side. If you can guide me a little through a cleaner solution, I'd be more than happy to implement it. @hdavidzhu

Reply:
> The only other option would be to create a class containing that css and to inject the class inside the 3 styled components, but I'm not sure if that's a very clean approach either.

That's what I meant by composition; isn't that what Emotion excels at? https://emotion.sh/docs/composition It outputs a new css class that would combine the modifiers of what's passed in

Reply:
Yup I'll open a PR for it. It's going to be like 4-5 classes and 3-4 styled components. I guess I see this as a double-edged sword. There's a benefit in having all of the table's styling in one place, and separating it apart, then combining it, could make the styling harder to understand. On the other hand, class names may help explaining in a friendlier manner what each piece of CSS actually does to the table, which I like.  

On a side note, I don't think composition is better/superior to simple SCSS nesting, which front-end engineers have used for a long time already. Just another way of writing CSS that we can use when it makes sense to do so.  I'll usually gear towards composition when I think something is going to be reused a lot in the future.

Reply:
#50